### PR TITLE
Added sample to Find-and-Replace-section-break-in-Word-document-with-page-break

### PR DESCRIPTION
**Pull request by elizabethatienoo**
Added sample of how to find and replace section break in Word document with page break

### CODE REVIEW

File: Find-and-Replace/Replace-section-break-with-page-break/.NET-Standard/Replace-section-break-with-page-break/Program.cs [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            using (FileStream fileStreamPath = new FileStream(Path.GetFullPath(@"../../../Data/Input.docx"), FileMode.Open, FileAccess.Read, FileShare.ReadWrite))
+            {
+                //Open an existing Word document.
+                using (WordDocument document = new WordDocument(fileStreamPath, FormatType.Docx))
+                {
+                    ReplaceSectionBreakWithPageBreak(document);
```
Comment (MohanaselvamJothi) [inline Find-and-Replace/Replace-section-break-with-page-break/.NET-Standard/Replace-section-break-with-page-break/Program.cs:16]: Add XML comment

File: Find-and-Replace/Replace-section-break-with-page-break/.NET-Standard/Replace-section-break-with-page-break/Program.cs [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+                }
+            }
+        }
+        /// <summary>
+        /// Replace the section break with page break in the Word document.
+        /// </summary>
+        /// <param name="document"></param>
+        private static void ReplaceSectionBreakWithPageBreak(WordDocument document)
+        {
+            //Add page break and removes section break by moving the section items to the last section.
```
Comment (MohanaselvamJothi) [inline Find-and-Replace/Replace-section-break-with-page-break/.NET-Standard/Replace-section-break-with-page-break/Program.cs:32]: Whether we are moving to last section or first section. Check and update XML comment

File: Find-and-Replace/Replace-section-break-with-page-break/.NET-Standard/Replace-section-break-with-page-break/Program.cs [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+            while (document.Sections.Count > 1)
+            {
+                WSection sec = document.Sections[1];
+                (document.Sections[0].Body.AddParagraph()).AppendBreak(BreakType.PageBreak);
```
Comment (MohanaselvamJothi) [inline Find-and-Replace/Replace-section-break-with-page-break/.NET-Standard/Replace-section-break-with-page-break/Program.cs:36]: Add XML comment

**Review by MohanaselvamJothi — CHANGES_REQUESTED** [review]

File: Find-and-Replace/Replace-section-break-with-page-break/.NET-Standard/Replace-section-break-with-page-break/Program.cs (hunk 2 above)
Comment (elizabethatienoo) [reply Find-and-Replace/Replace-section-break-with-page-break/.NET-Standard/Replace-section-break-with-page-break/Program.cs:32]: First section

File: Find-and-Replace/Replace-section-break-with-page-break/.NET-Standard/Replace-section-break-with-page-break/Program.cs (hunk 3 above)
Comment (elizabethatienoo) [reply Find-and-Replace/Replace-section-break-with-page-break/.NET-Standard/Replace-section-break-with-page-break/Program.cs:38]: Done

File: Find-and-Replace/Replace-section-break-with-page-break/.NET-Standard/Replace-section-break-with-page-break/Program.cs (hunk 1 above)
Comment (elizabethatienoo) [reply Find-and-Replace/Replace-section-break-with-page-break/.NET-Standard/Replace-section-break-with-page-break/Program.cs:17]: Addressed

File: Find-and-Replace/Replace-section-break-with-page-break/.NET-Standard/Replace-section-break-with-page-break/Program.cs [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+                        //Save the Word document to file stream.
+                        document.Save(outputFileStream, FormatType.Docx);
+                    }
+                }
+            }
+        }
+        /// <summary>
+        /// Replace the section break with page break in the Word document.
+        /// </summary>
+        /// <param name="document"></param>
```
Comment (MohanaselvamJothi) [inline Find-and-Replace/Replace-section-break-with-page-break/.NET-Standard/Replace-section-break-with-page-break/Program.cs:30]: Please remove this line

**Review by MohanaselvamJothi — CHANGES_REQUESTED** [review]

**Review by MohanaselvamJothi — APPROVED** [review]

**Review by MohanaselvamJothi — APPROVED** [review]

**Pull request merged**